### PR TITLE
Stardardize endpoints and publish other CRUD operations futher than Add

### PR DESCRIPTION
**Pull request by Odrakir1**

### CODE REVIEW

File: modules/amf-registration/amf-registration-web-rest-js/amf-registration-web-rest/src/main/java/com/acme/registration/web/AcmeAccountRegistrationApplication.java [hunk 1]
```diff
@@ -129,6 +134,33 @@ public Response getAccount(
...
+		}
+
+		try {
+			String json = _OBJECT_MAPPER.writeValueAsString(accountsDTO);
+
+			return Response.ok(
+					json, MediaType.APPLICATION_JSON
+			).build();
+		}
+		catch (JsonProcessingException jPE) {
```
Comment (matheusleon) [inline modules/amf-registration/amf-registration-web-rest-js/amf-registration-web-rest/src/main/java/com/acme/registration/web/AcmeAccountRegistrationApplication.java:166]: @Odrakir1 I think the source formatter will complain about this. It should be `JsonProcessingException jsonProcessingException.`

**Comment by alexandresjunior** [comment]
hey, guys! nice work, thanks! can you warn me or @arthurmenezesb before merging branches? we'd like to keep track on those PR's before merging them. thank you! by the way, @Odrakir1 good job at #8 o/

File: modules/amf-registration/amf-registration-web-rest-js/amf-registration-web-rest/src/main/java/com/acme/registration/web/AcmeAccountRegistrationApplication.java (hunk 1 above)
Comment (Odrakir1) [reply modules/amf-registration/amf-registration-web-rest-js/amf-registration-web-rest/src/main/java/com/acme/registration/web/AcmeAccountRegistrationApplication.java:166]: Hey @mlan0, thank you for this reminder, I've ran the SF, it changed some lines, but it didn't complain about this. Could you please take a look at [this link for reference](https://github.com/liferay/liferay-portal/search?q=ParseException+pe)? It seems to be optional using the whole exception name or its initials. Thank you!

**Comment by Odrakir1** [comment]
Hey, @alexandresjunior! Thank you for your support! I'll keep this in mind for the next time! o/

**Review by matheusleon — APPROVED** [review]

**Comment by brunoquei2105** [comment]
LGTM

File: modules/amf-registration/amf-registration-web-rest-js/amf-registration-web-rest/src/main/java/com/acme/registration/dto/AccountDTO.java [hunk 2]
```diff
@@ -0,0 +1,333 @@
+package com.acme.registration.dto;
+
+import com.acme.registration.model.Account;
+
```
Comment (brunoquei2105) [inline modules/amf-registration/amf-registration-web-rest-js/amf-registration-web-rest/src/main/java/com/acme/registration/dto/AccountDTO.java:4]: Everything's fine.

**Review by brunoquei2105 — APPROVED** [review]

**Pull request merged**